src/doc/rustc/src/platform-support.md

```diff
@@ -384,7 +384,10 @@ target | std | host | notes
 [`x86_64h-apple-darwin`](platform-support/x86_64h-apple-darwin.md) | ✓ | ✓ | macOS with late-gen Intel (at least Haswell)
 [`x86_64-unknown-linux-none`](platform-support/x86_64-unknown-linux-none.md) | * |  | 64-bit Linux with no libc
 `xtensa-esp32-none-elf` |  |  | Xtensa ESP32
+`xtensa-esp32-espidf` |  |  | Xtensa ESP32
 `xtensa-esp32s2-none-elf` |  |  | Xtensa ESP32-S2
+`xtensa-esp32s2-espidf` |  |  | Xtensa ESP32-S2
 `xtensa-esp32s3-none-elf` |  |  | Xtensa ESP32-S3
+`xtensa-esp32s3-espidf` |  |  | Xtensa ESP32-S3
```

Comment:
Could you also fill out the `std` column? Presumably this should be `*` because the target only supports `no_std`?

Reply:
Good catch! `xtensa-*-none-elf` support only `no_std`, while `xtensa-*-espidf` supports most of `std`. Updated!